defaults/main.yml

```diff
@@ -65,3 +65,18 @@ aem_cms_in_place_upgrade_paths:
     - "6.4.0"
 # "from_version":
 #    - "to_version" # list of versions that an upgrade is allowed for
+
+# Seconds to wait for instance to be stopped until process is killed
+aem_cms_stop_timeout_seconds: 180
```

Comment:
Hmm, this is closely related to `aem_service_timeout`, which makes it a bit confusing to configure.
Also, 180s (compared to the 1200 in aem-service) is a bit on the short side. A huge instance might take considerably longer to shutdown and would be killed by this default.
In fact, since the killing part is kind of dangerous I think it should probably be optional.

Reply:
This was increased to 1200 seconds.
I also requested a change in https://github.com/wcm-io-devops/conga-aem-definitions/pull/43 even when this role does not depende on them.